docs/index.md

```diff
@@ -16,8 +16,9 @@ More details on
 **Googlers only** : clangd is installed on `/usr/bin/clangd` by default on
 glinux workstations, you can directly use that instead.
 
-[clangd/releases](https://github.com/clangd/clangd/releases) are
-built with that support for major platforms. You can find out about other
+Remote index support requires **clangd 12** or newer, and is not yet included
```

Comment:
can we elaborate a little:
```
Remote index requires at least clangd-12 build with `CLANGD_ENABLE_REMOTE` option on. 
You can verify that by running `clangd --version` and checking for `grpc` in the `Features` line.

[clangd/releases]....
```

Reply:
hmm, the CLANGD_ENABLE_REMOTE suggestion can be misleading unless we also tell the reader what additional dependencies are needed (grpc, protobuf, …). i think it would be better to point the reader to the upstream docs.

also, the “Features” suggestion only works on clangd 13, not clangd 12, even when built with remote index support.

Reply:
i’ve tried to address these in 6f96b5b9f36de016d9bcf6feaa4443374d27ca1f, what do you think?